packages/itinerary-body/src/otp-react-redux/view-stop-button.tsx

```diff
@@ -5,15 +5,23 @@ import { FormattedMessage } from "react-intl";
 import * as S from "../styled";
 import { defaultMessages } from "../util";
 
-interface Props {
+interface PropsWithStopId {
```

Comment:
I think you should be able to combine these two types, with the check for `"gtfsId" in stop` be sufficient. 
```
interface Props {
  onStopClick: ...;
  stopId?: string;
  stop?: Stop;
}
```